### PR TITLE
feat: add release notes / changelog

### PR DESCRIPTION
**Pull request by s0up4200**

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ This isn't needed if your use case is feeding the \*arrs only. Since they have R
 
 As of right now, autobrr features:
 
-- Support for 75+ trackers with IRC announces
+- Support for 70+ trackers with IRC announces
```
Comment (martylukyy) [inline docs/introduction.md:100]: This is outdated information - revert the changes on this file.

**Comment by martylukyy** [comment]
The dates shown on the release note pages are off..
Idk if you care about that but if you do maybe look them up on the releases page on GitHub.

File: docs/introduction.md (hunk 1 above)
Comment (s0up4200) [reply docs/introduction.md]: Strange. Had to manually change it, even after merging main into the branch. Probably because it was newer or something. Fixed.

**Comment by s0up4200** [comment]
> The dates shown on the release note pages are off.. Idk if you care about that but if you do maybe look them up on the releases page on GitHub.

I took the dates from Discord, which seems to be correct for all of them except for v.1.30. Fixed!
Clearly shows that z needs to post release notes faster 😉

**Review by martylukyy — APPROVED** [review]
